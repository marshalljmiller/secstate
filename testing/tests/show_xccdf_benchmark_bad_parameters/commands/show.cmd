/usr/bin/secstate show PassComps
