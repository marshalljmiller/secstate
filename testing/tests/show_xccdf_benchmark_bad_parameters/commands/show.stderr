Item 'PassComps' could not be found in the imported content
